md/taxonomy.md

```diff
@@ -209,6 +209,7 @@
     * [Create a new Look & Feel](creating-a-new-look-feel.md)
   * [Search keys](search-index.md)
 * [Bonita Runtime Monitoring](runtime-monitoring.md)
```

Comment:
⚠️ This overwrites the real content: see PR doc page: https://cidoc.cloud.bonitasoft.com:21279/bonita/7.11/runtime-monitoring => it does not show the MD content, instead of https://github.com/bonitasoft/bonita-doc/blob/493a394d243efe019de42ac535c573a5efa50a0d/md/runtime-monitoring.md !!!
Is it because it has **sub-pages** ?

Reply:
Will fix, thanks